src/stem/settings.py

```diff
@@ -147,6 +153,11 @@
 	'DEFAULT_PERMISSION_CLASSES': [
         'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly'
     ],
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework_jwt.authentication.JSONWebTokenAuthentication',
+        'rest_framework.authentication.SessionAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
```

Comment:
Do we need `BasicAuthentication` and `SessionAuthentication` here? I'm curious if we only need `JSONWebTokenAuthentication` for this setup.

Reply:
I'm not sure what the best practice is, but I removed them from my project from last semester.

Reply:
I am still playing around with JWT. Yeah, we don't need them. I will remove them.